src/features/ProfilePage/components/AccountInfo.tsx

```diff
@@ -27,14 +26,24 @@ type Props = {
 
 const AccountInfo = ({ isMobile = false }: Props) => {
   const { t } = useTranslation('profile');
+  const { getConfirmation } = useConfirm();
   const { id, login_name: loginName, role } = useAppSelector(selectAccount);
   const isMentor = useAppSelector(selectIsMentor);
   const [deleteAccount] = useDeleteAccountMutation();
 
-  const [isDeleteConfirmationOpen, setIsDeleteConfirmationOpen] =
-    useState(false);
-  const openDeleteConfirmation = () => setIsDeleteConfirmationOpen(true);
-  const closeDeleteConfirmation = () => setIsDeleteConfirmationOpen(false);
+  const confirmDelete = async () => {
```

Comment:
Here too I would just rename the function to `deleteAccount`

Reply:
It can't be `deleteAccount`, because we already have the mutation with that name.